### PR TITLE
Implement `TagPanel` component

### PR DESCRIPTION
**Pull request by alexeykostevich**
Fixes https://github.com/ember-learn/empress-blog-ember-template/issues/31.

Screen Shot 2020-05-13 at 1 30 27 am

### CODE REVIEW

**Comment by alexeykostevich** [comment]
Please, do not merge the PR since it does not include integration tests — they will be added shortly.

The PR has been created for early feedback.

**Comment by alexeykostevich** [comment]
@mansona, please, pay attention that the component renders all the titles in Title Case. This does not fully match [the wireframes](https://github.com/ember-learn/empress-blog-ember-template/issues/31), which have minor inconsistencies among different tag groups.

Please, let me know if you would like to change this.

File: addon/components/tag-panel.hbs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      >
+        By Year
+      </h3>
+
+      <ul
+        class="tag-panel-inline-group list-unstyled ml-1"
+        aria-labelledby="year-tag-group-title"
+        data-test-tag-list
+      >
+        {{#each this.yearTags key="id" as |yearTag|}}
```
Comment (mansona) [inline addon/components/tag-panel.hbs:35]: you don't need to pass key here, it's automatic 👍

File: addon/components/tag-panel.hbs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      >
+        By Major Version
+      </h3>
+
+      <ul
+        class="tag-panel-inline-group list-unstyled ml-1"
+        aria-labelledby="version-tag-group-title"
+        data-test-tag-list
+      >
+        {{#each this.versionTags key="id" as |versionTag|}}
```
Comment (mansona) [inline addon/components/tag-panel.hbs:65]: same again, you can remove key 👍

File: addon/components/tag-panel.hbs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      >
+        By Subject
+      </h3>
+
+      <ul
+        class="list-unstyled ml-1"
+        aria-labelledby="subject-tag-group-title"
+        data-test-tag-list
+      >
+        {{#each this.subjectTags key="id" as |subjectTag|}}
```
Comment (mansona) [inline addon/components/tag-panel.hbs:95]: remove key 👍

**Comment by mansona** [comment]
@alexeykostevich I've done a quick review 👍 

you mention that you would prefer that this wasn't merged 🤔 firstly, are you not able to create a "draft PR" on this repo? or are you not able to convert this back into a draft PR? Secondly, I'm really quite happy to merge this as is, and if you wanted to add some testing then that could be a following PR but it wouldn't strictly be required.

I'm not going to merge this without you being ok with it though 😄

File: addon/components/tag-panel.hbs (hunk 1 above)
Comment (alexeykostevich) [reply addon/components/tag-panel.hbs]: Great catch 👍 Fixed!

File: addon/components/tag-panel.hbs (hunk 2 above)
Comment (alexeykostevich) [reply addon/components/tag-panel.hbs]: Great catch 👍 Fixed!

File: addon/components/tag-panel.hbs (hunk 3 above)
Comment (alexeykostevich) [reply addon/components/tag-panel.hbs]: Great catch 👍 Fixed!

**Comment by alexeykostevich** [comment]
Oh, frankly, I just didn't know about draft PRs 🙂 Thank you for sharing and drawing my attention to this, @mansona — this is very handy!

I've converted the PR to draft and started working on integration tests 🙂

**Pull request merged**